.buildkite/pipeline.yml

```diff
@@ -5,7 +5,7 @@ steps:
     concurrency: 1
     concurrency_group: primary
     timeout_in_minutes: 45
-    branches: master
+    branches: "master 0.10.x"
```

Comment:
Is this the correct syntax? I thought it was an array, not space delimited

Reply:
I was surprised to, but according to https://buildkite.com/docs/pipelines/branch-configuration#branch-pattern-examples it is a space separated string